vivarium_models/processes/simularium_emitter.py

```diff
@@ -68,13 +67,21 @@ def get_simularium_monomers(self, time, monomers, actin_radius, trajectory):
         trajectory["unique_ids"].append([])
         trajectory["type_names"].append([])
         trajectory["positions"].append([])
+        trajectory["radii"].append([])
         edge_ids = []
         edge_positions = []
         for particle_id in monomers["particles"]:
             particle = monomers["particles"][particle_id]
             trajectory["unique_ids"][time_index].append(int(particle_id))
             trajectory["type_names"][time_index].append(particle["type_name"])
+            # HACK needed until simulariumio default display data fix
```

Comment:
I thought the default display data fix got merged in simulariumio, unless there's another default display data bug than the one I'm thinking of?